stack-snapshot.yaml

```diff
@@ -1,9 +1,8 @@
 # Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-resolver: nightly-2022-01-26
+resolver: lts-19.6
```

Comment:
not quite sure I understand why this has to change at least in this PR. Not opposed to the change though (although I tend to prefer nightly personally but definitely don’t view my opinion as a blocker here).

Reply:
oh, for some reason I thought it would be better to have all snapshots on the same package set, but it should be enough for them to work with the same ghc version. I'll revert the changes here

Reply:
I do think this is a sensible goal. I guess I’m more asking about the choice of snapshot than about keeping them consistent. If you do want to bump it maybe worth factoring it out (you’ll also have to bump for windows).